<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>2048</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(251, 248, 239);</string>
  </property>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>50</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/icon.jpeg);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblAge">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>60</y>
     <width>60</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/age.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>75</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnName">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>170</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>修改用户名</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/interface/images/changename.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblWarning">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>340</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>健康游戏忠告
抵制不良游戏，拒绝盗版游戏。
注意自我保护，谨防受骗上当。
适度游戏益脑，沉迷游戏伤身。
合理安排时间，享受健康生活。</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ldtName">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);
border: 1px solid gray;
border-radius: 5px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>enter new name</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnPlayer">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	border-image: url(:/interface/images/people.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnGift">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	border-image: url(:/interface/images/gift.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRank">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>385</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	border-image: url(:/interface/images/rank.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnPassword">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>385</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	border-image: url(:/interface/images/password.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRet">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>退出登录</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/interface/images/home.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayer">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>325</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>管理用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRank">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblGift">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>325</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>礼包码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>470</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSystem">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>325</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnSystem">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>240</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	border-image: url(:/interface/images/system.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblHistory">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>用户历史</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnHistory">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>385</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	border-image: url(:/interface/images/history.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>ldtName</zorder>
  <zorder>lblName</zorder>
  <zorder>lblIcon</zorder>
  <zorder>lblAge</zorder>
  <zorder>lblTitle</zorder>
  <zorder>pBtnName</zorder>
  <zorder>lblWarning</zorder>
  <zorder>pBtnPlayer</zorder>
  <zorder>pBtnGift</zorder>
  <zorder>pBtnRank</zorder>
  <zorder>pBtnPassword</zorder>
  <zorder>pBtnRet</zorder>
  <zorder>lblPlayer</zorder>
  <zorder>lblRank</zorder>
  <zorder>lblGift</zorder>
  <zorder>lblPassword</zorder>
  <zorder>lblSystem</zorder>
  <zorder>pBtnSystem</zorder>
  <zorder>lblHistory</zorder>
  <zorder>pBtnHistory</zorder>
 </widget>
 <tabstops>
  <tabstop>pBtnName</tabstop>
  <tabstop>ldtName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
